### ISSUE TITLE
Playground: optionally highlight non-LD JSON parts 

### ISSUE THREAD

**Issue by neothemachine** [json-ld/json-ld.org#393]
I often hack around in JSON-LD in the playground and it happens that I forget something in the context or have a typo in it. While this is easy to see for very small documents in the output, it is harder for bigger ones. I thought that it would be really handy if you could tick a checkbox which then would highlight all properties in red which will not result in RDF triples. The rest could be highlighted in green. Is that hard to implement?

**Comment by msporny** [comment]
Yes, that would be an awesome feature. Yes, it's also hard to implement. We'd love someone to work on it, we just don't have the spare bandwidth right now to implement those types of features in the playground.
